superset3/DockerFile.baseImage.template

```diff
@@ -1,43 +1,33 @@
-FROM apache/superset:3.1.3
+# Use the specific Superset base image
+FROM {{BASE_IMAGE}}
 
 # Switching to root to install the required packages
 USER root
 
-# https://superset.apache.org/docs/installation/alerts-reports/#custom-dockerfile
+# Install system packages
 RUN apt-get update && \
   apt-get install --no-install-recommends -y firefox-esr wget
 
+# Set the version for Geckodriver
 ENV GECKODRIVER_VERSION=0.29.0
+
+# Install Geckodriver
 RUN wget -q https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VERSION}/geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz && \
   tar -x geckodriver -zf geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz -O > /usr/bin/geckodriver && \
   chmod 755 /usr/bin/geckodriver && \
   rm geckodriver-v${GECKODRIVER_VERSION}-linux64.tar.gz
 
-# python packages
-USER root
+# Upgrade pip
```

Comment:
you removed `USER root`? does it still work?

Reply:
> you removed `USER root`? does it still work?

yes, we already have USER root at the top level and we are only changin it back to superset user at the end, so it works.